test/functional/element_creation.js

```diff
@@ -109,6 +110,58 @@ describe('element creation', () => {
 
   });
 
+
+  describe('should return getCurrentNode', () => {
```

Comment:
It seems that this should probably be a top level `describe` (it its own file) since currentElement is really it's own feature that also applies on updates not just element creation.